components/Ui/UiCompactFeature.vue

```diff
@@ -50,8 +62,9 @@ $feature-icon-size: 2.25rem;
     }
   }
 
-  &__description {
+  &__copy {
     color: qiskit.$text-color-white;
+    margin-bottom: carbon.$spacing-05;
```

Comment:
![Screenshot 2023-08-03 at 17 50 29](https://github.com/Qiskit/qiskit.org/assets/22047320/0305b388-13ae-4084-b206-a9e826272792)

I think that this might add too much space at the end of the card. Maybe the best approach here would be to only add a margin-top to the "detail" div.

Reply:
Nicely implemented in https://github.com/Qiskit/qiskit.org/pull/3477/commits/d1108671665325a9056820f665cacccd06c0665b, thank you!